helm/charts/hydra-maester/README.md

```diff
@@ -25,6 +25,7 @@ A Helm chart for Kubernetes
 | deployment.tolerations | list | `[]` | k8s toleration |
 | enabledNamespaces | list | `[]` | The Controller have CREATE and READ access to all Secrets in the namespaces listed below. |
 | forwardedProto | string | `nil` |  |
+| syncPeriod | string | `10h` | The minimum frequency at which watched resources are reconciled |
```

Comment:
This file is generated by https://github.com/norwoodj/helm-docs, just wanted to make sure you didn't modify it manually ;)

Reply:
ah, I did! I'll add the comment to the Values.yml file and leave the readme alone then.

Reply:
Is it intended that I run the tool to generate the readme and include it in this PR?
When I run helm-docs I get a very sparse readme missing a lot of the text.

Reply:
I see, it seems in some release of the tool, something in the description changed, right now it requires syntax like 
```
# -- The Controller have CREATE and READ access to all Secrets in the namespaces listed below.
enabledNamespaces: []
```
seems the `--` is required. I will create a issue for fixing that globally :)